yt_dlp/extractor/globo.py

```diff
@@ -67,11 +67,26 @@ class GloboIE(InfoExtractor):
     }, {
         'url': 'globo:3607726',
         'only_matching': True,
+    }, {
+        'url': 'https://globoplay.globo.com/v/10248083/',
+        'info_dict': {
+            'id': '10248083',
+            'ext': 'mp4',
+            'title': 'Melhores momentos: Equador 1 x 1 Brasil pelas Eliminatórias da Copa do Mundo 2022',
+            'duration': 530.964,
+            'uploader': 'SporTV',
+            'uploader_id': '698',
+        },
+        'params': {
+            'skip_download': True,
+        },
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
+        self._download_webpage('https://globo-ab.globo.com/v2/selected-alternatives?experiments=player-isolated-experiment-02&skipImpressions=true', video_id, 'Getting cookies')
```

Comment:
try this
```suggestion
        self._request_webpage(
            HEADRequest('https://globo-ab.globo.com/v2/selected-alternatives?experiments=player-isolated-experiment-02&skipImpressions=true'),
            video_id, 'Getting cookies')
```

Reply:
with this changes flake8 is returning errors.

> D:\ytdlp\yt-dlp>flake8 yt_dlp\extractor\globo.py
yt_dlp\extractor\globo.py:89:13: F821 undefined name 'HEADRequest'
yt_dlp\extractor\globo.py:112:31: F821 undefined name 'HeadRequest'